main.go

```diff
@@ -161,6 +163,11 @@ func tagCommand() *cli.Command {
 				Value:       cli.NewStringSlice(utils.GetAllTagGroupsNames()...),
 				DefaultText: "git,code2cloud",
 			},
+			&cli.StringFlag{
```

Comment:
@tronxd This is a workaround, it doesn't leverage the lib's [inner implementation of yaml loading](https://github.com/urfave/cli/blob/master/docs/v2/manual.md#values-from-alternate-input-sources-yaml-toml-and-others). Can we leverage that instead of writing all the yaml parsing ourselves?

Reply:
@nimrodkor there is no parsing except what we want to parse internally (no YAML lib can parse more than just unmarshaling to a scheme

Reply:
@nimrodkor I will use this option only when binding it to the respected new CLI options